app/views/users/show.html.erb

```diff
@@ -1,2 +1,21 @@
-<h1>Single User</h1>
-<p>This is a placeholder for a single user.</p>
+<div class="jumbotron shadow p-3">
+  <%= image_tag(@user.photo, alt: @user.name, class: 'user-image') %>
+  <h2 class="display-2 font-weight-bold mt-2 d-inline-block"><%= @user.name %></h2>
+  <p class="right">Number of posts: <%= @user.posts.count %></p>
+</div>
+
+<div class="jumbotron shadow p-3">
+  <p>Bio</p>
+  <p><%= @user.bio %></p>
+</div>
+
+<% @user.posts.each_with_index do |post, index| %>
+  <div class="jumbotron p-3 shadow">
+    <p>Post#<%= index + 1 %></p>
+    <p><%= post.title %></p>
+  </div>
+<% end %>
```

Comment:
- [ ] Good job so far 👏 However, kindly make sure to use the `#recent_posts` method to render the user's 3 most recent post mentioned in the [project requirements] and according to the provided wireframe as well 🤓⛏
- [ ] Also, please make sure to display the post text as we have post title. Currently, only title is visible, let's also add the text ✨💪 something like below 👇


```ruby
<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
```

- [ ] Also, Kindle make sure to display the comments counter and likes counter associated with each post as displayed on the wireframe ✨ you can use something like below 👇

```ruby
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>
```

Current UI | Wireframe
---| ---|
![image](https://github.com/jkanyi-web/Blog-App/assets/106119105/eb2b82bb-d486-430a-93d0-3b9a6e854a1e) | ![image](https://github.com/jkanyi-web/Blog-App/assets/106119105/19392388-f109-40ea-bb2d-570148d55250)